src/kili/use_cases/cloud_storage/__init__.py

```diff
@@ -303,3 +306,60 @@ def _validate_data_differences(
                     f" {nb_assets_before} assets, after: {nb_assets_after} assets,"
                     f" dataDifferencesSummary: {asset_diff}"
                 )
+
+
+def create_data_integration(
+    self,
+    allowed_paths: Optional[List[str]],
+    aws_access_point_arn: Optional[str],
+    aws_role_arn: Optional[str],
+    aws_role_external_id: Optional[str],
+    azure_connection_url: Optional[str],
+    azure_is_using_service_credentials: Optional[bool],
+    azure_sas_token: Optional[str],
+    azure_tenant_id: Optional[str],
+    gcp_bucket_name: Optional[str],
+    include_root_files: Optional[str],
+    internal_processing_authorized: Optional[str],
+    name: str,
+    platform: DataIntegrationPlatform,
+    s3_access_key: Optional[str],
+    s3_bucket_name: Optional[str],
+    s3_endpoint: Optional[str],
+    s3_region: Optional[str],
+    s3_secret_key: Optional[str],
+    s3_session_token: Optional[str],
+    fields: ListOrTuple[str],
+) -> Dict:
+    """Create a data integration."""
+    data = DataIntegrationData(
+        author_id=None,
+        allowed_paths=allowed_paths,
+        aws_access_point_arn=aws_access_point_arn,
+        aws_role_arn=aws_role_arn,
+        aws_role_external_id=aws_role_external_id,
+        azure_connection_url=azure_connection_url,
+        azure_is_using_service_credentials=azure_is_using_service_credentials,
+        azure_sas_token=azure_sas_token,
+        azure_tenant_id=azure_tenant_id,
+        gcp_bucket_name=gcp_bucket_name,
+        include_root_files=include_root_files,
+        internal_processing_authorized=internal_processing_authorized,
+        name=name,
+        organization_id=OrganizationId(""),
```

Comment:
I guess it is ignored backend side but kept required in the GraphQL schema?

Reply:
indeed